python/py-gnureadline/Portfile

```diff
@@ -8,7 +8,7 @@ version             8.0.0
 platforms           darwin
 license             GPL-3+ PSF
 
-python.versions     27 34 35 36 37 38
+python.versions     27 35 36 37 38
```

Comment:
Isn't python 27 itself obsolete now?

Reply:
Heh. Sure. Apart from the upcoming maintenance release. And the fact that everyone isn't going to just stop needing it because it's been declared EOL.

Reply:
What maintenance release? As of Jan 1, there will be no further maintenance of Python 2, full stop, or so everything seems to indicate.

And yes, we probably need to support it a bit going forward, but I would encourage us to figure out how to sunset support.

Reply:
https://www.python.org/dev/peps/pep-0373/#maintenance-releases

Reply:
> Support officially stops January 1 2020, but the final release will occur after that date.

Okay, so I suppose that there will be a future release. _But_, support is over.